CMakeLists.txt

```diff
@@ -87,13 +70,25 @@ if(APPLE)
     endif()
     message(STATUS "Using MoltenVK repo location at ${MOLTENVK_REPO_ROOT}")
 endif()
-message(STATUS "Using find_package to locate Vulkan")
-find_package(Vulkan)
+
 find_package(VulkanHeaders REQUIRED CONFIG)
-get_filename_component(Vulkan_LIBRARY_DIR ${Vulkan_LIBRARY} DIRECTORY)
-message(STATUS "Vulkan FOUND = ${Vulkan_FOUND}")
-message(STATUS "Vulkan Lib Dir = ${Vulkan_LIBRARY_DIR}")
-message(STATUS "Vulkan Lib = ${Vulkan_LIBRARY}")
+
+# Find and create the Vulkan::Vulkan library
+if (WIN32)
+    find_program(Vulkan_LIBRARY NAMES vulkan-1.dll)
```

Comment:
Since this code is only needed for testing move it into `tests/CMakeLists.txt`

Reply:
Do you mean the creating of `Vulkan::Vulkan` entirely, or just the parts that find `vulkan-1.dll` specifically? I am 99% sure that vkcube links to `vulkan-1.lib` so that part is necessary even when tests aren't being built.

Reply:
just the parts that find vulkan-1.dll specifically